src/frontend/js/draftjs-plugins/code-editor/index.ts

```diff
@@ -0,0 +1,95 @@
+/**
+ * Credits :
+ * Forked from https://github.com/withspectrum/draft-js-code-editor-plugin
+ * - added an unstyle block after key return is pressed. This is done by inserting a new
+ *   block when the split-block keyCommand is fired.
+ */
+
+import CodeUtils from 'draft-js-code';
+import Draft, {
+  ContentBlock,
+  EditorState,
+  KeyBindingUtil,
+  Modifier,
+  RichUtils,
+} from 'draft-js';
+import { PluginFunctions } from 'draft-js-plugins-editor/lib';
+import React from 'react';
+import { List } from 'immutable';
+type Options = {};
+type SyntheticKeyboardEvent = React.KeyboardEvent<{}>;
+
+const createCodeEditorPlugin = (options?: Options) => {
+  return {
+    handleKeyCommand(
+      command: string,
+      editorState: EditorState,
+      eventTimeStamp: number,
+      { setEditorState }: PluginFunctions,
+    ) {
+      let newState;
+
+      if (CodeUtils.hasSelectionInBlock(editorState)) {
+        newState = CodeUtils.handleKeyCommand(editorState, command);
+      } else {
+        newState = RichUtils.handleKeyCommand(editorState, command);
+      }
```

Comment:
```js
const newState = CodeUtils.hasSelectionInBlock(editorState)
  ? CodeUtils.handleKeyCommand(editorState, command)
  : RichUtils.handleKeyCommand(editorState, command)
```

In this way you are able to use a constant that is more suitable here.

Reply:
Hi JB, thanks for you feedback, I've changed it this way !